tasks/gateway.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Include initial OS-specific tasks
+  ansible.builtin.include_tasks: init_{{ ansible_os_family | lower }}.yml
+  vars:
+    _cvmfs_role: gateway
+    _cvmfs_upgrade: "{{ cvmfs_upgrade_server }}"
+
+- name: Create gateway API keys for repository publishing access (stratum 0)
+  template:
+    src: repos.json.j2
+    dest: "/etc/cvmfs/gateway/repo.json"
+    force: no
+    owner: root
+    group: root
+    mode: '644'
+  vars:
+    cvmfs_repo_name: "{{ item.key }}"
+    cvmfs_gateway_key: "{{ lookup('password', '/dev/null chars=ascii_letters,digits') }}" 
+  with_dict: "{{ cvmfs_repo_list }}"
```

Comment:
I can't find the documentation for this variable. Is this a new variable introduced with the change?

Reply:
fixed